src/System.Runtime.Extensions/tests/System/AppDomainTests.cs

```diff
@@ -2,39 +2,33 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System;
 using System.Diagnostics;
 using System.IO;
 using System.Reflection;
 using System.Resources;
 using System.Runtime.ExceptionServices;
 using Xunit;
-using Xunit.NetCore.Extensions;
 
 namespace System.Tests
 {
     // No appdomain in UWP or CoreRT
     [SkipOnTargetFramework(TargetFrameworkMonikers.UapAot | TargetFrameworkMonikers.NetFramework, "dotnet/corefx #18718")]
```

Comment:
this may fix https://github.com/dotnet/corefx/issues/18718 and tes tcan be enabled for desktop?